manifests/2.6.0/opensearch-2.6.0.yml

```diff
@@ -140,7 +140,7 @@ components:
       - gradle:dependencies:opensearch.version
   - name: anomaly-detection
     repository: https://github.com/opensearch-project/anomaly-detection.git
-    ref: 2.x
+    ref: tags/2.6.0.0
```

Comment:
It wont be tag until 2.6.0.0 is released. Did you mean to change it to `2.6` branch?